plugin/hop.vim

```diff
@@ -12,7 +12,7 @@ command! HopWord lua require'hop'.hint_words()
 command! HopPattern lua require'hop'.hint_patterns()
 
 " The jump-to-char-1 command.
-command! HopChar1 lua require'hop'.hint_char1()
+command! HopChar1 lua require'hop'.hint_char1(ANYWHERE, false)
```

Comment:
I don’t think those constants are really adding value, since they are aliases to strings which value is the same as the constant name. Also, I think this line is wrong and you should be using `require'hop.hint'.ANYWHERE'` instead, which is not currently going to work as those constants are not added to the returned `M` object.

I think it’s okay to let people use literal strings here.